corehq/apps/users/models_sql.py

```diff
@@ -228,6 +233,41 @@ class RoleAssignableBy(models.Model):
     )
 
 
+class UpdateDetails(jsonobject.JsonObject):
+    changed_via = jsonobject.StringProperty()
+    changes = jsonobject.DictProperty()
+
+
+class HQLogEntry(models.Model):
+    """
+    HQ Adaptation of Django's LogEntry model
+    """
+    CREATE = 1
+    UPDATE = 2
+    DELETE = 3
+
+    ACTION_CHOICES = (
+        (CREATE, _('Create')),
+        (UPDATE, _('Update')),
+        (DELETE, _('Delete')),
+    )
+    domain = models.CharField(max_length=255, null=True)
+    object_type = models.CharField(max_length=255)
+    object_id = models.CharField(max_length=128)
+    by_user_id = models.CharField(max_length=128)
+    details = JSONField(default=dict)  # UpdateDetails
+    message = models.TextField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    action = models.PositiveSmallIntegerField(choices=ACTION_CHOICES)
+
+    class Meta:
+        indexes = [
+            models.Index(fields=['domain']),
+            models.Index(fields=['object_id'])
+        ]
+        index_together = ["domain", "by_user_id"]
```

Comment:
why is this a combined index and the other one isn't? You're effectively creating two `domain` indexes.